### DISCUSSION TITLE
Rebuild model from events

### DISCUSSION THREAD

**Discussion by jacklam718** [looplab/eventhorizon#357] [Q&A] [answered]
How to replay all events and rebuild the read model from scratch?

**Comment by maxekman** [comment, marked as answer]
Currently you have to manually wire up the components to do that. It is possible that this will be added in the future.

**Comment by maxekman** [comment]
Unfortunately I don't have an example ready to show.

**Comment by jacklam718** [comment]
Thanks! Wish I can see the example soon. i spend a lot of time thinking about that and search relevant resources still have no idea.
